plugins/module_utils/sops.py

```diff
@@ -316,7 +318,7 @@ def get_sops_runner_from_options(get_option_value, module=None, display=None):
 
     @staticmethod
     def decrypt(encrypted_file, content=None,
-                display=None, decode_output=True, rstrip=True, input_type=None, output_type=None, get_option_value=None, module=None):
+                display=None, decode_output=True, rstrip=True, input_type=None, output_type=None, get_option_value=None, extract=None, module=None):
```

Comment:
Please add new arguments at the end:
```suggestion
                display=None, decode_output=True, rstrip=True, input_type=None, output_type=None, get_option_value=None, module=None, extract=None):
```
Otherwise you potentially break backwards compatibility.